tests/sources/xml/test_ead.py

```diff
@@ -1008,6 +1008,311 @@ def test_get_identifiers_transforms_correctly_if_type_attribute_invalid():
+                </extent>
+            </physdesc>
+            """
+        ),
+        parent_element="did",
+    )
+    assert Ead.get_physical_description(source_record) == (
+        "4.5 Cubic Feet (10 manuscript boxes, 1 legal manuscript box, 1 cassette box)"
+    )
+
+
+def test_get_physical_description_transforms_correctly_if_fields_blank():
+    source_record = create_ead_source_record_stub(
+        metadata_insert=(
+            """
+            <physdesc></physdesc>
+            """
+        ),
+        parent_element="did",
+    )
+    assert Ead.get_physical_description(source_record) is None
+
+
+def test_get_physical_description_transforms_correctly_if_fields_missing():
+    source_record = create_ead_source_record_stub(parent_element="did")
+    assert Ead.get_physical_description(source_record) is None
+
+
+def test_get_physical_description_transforms_correctly_if_multiple_physdesc():
+    source_record = create_ead_source_record_stub(
+        metadata_insert=(
+            """
+            <physdesc>
+                <extent>4.5 Cubic Feet</extent>
+                <extent>
+                    (10 manuscript boxes, 1 legal manuscript box, 1 cassette box)
+                </extent>
+            </physdesc>
+            <physdesc>
+                <extent>1.5 Cubic Feet</extent>
+                <extent>(2 manuscript boxes)</extent>
+            </physdesc>
+            """
+        ),
+        parent_element="did",
+    )
+    assert Ead.get_physical_description(source_record) == (
+        "4.5 Cubic Feet (10 manuscript boxes, 1 legal manuscript box, 1 cassette box); "
+        "1.5 Cubic Feet (2 manuscript boxes)"
+    )
+
+
+def test_get_publishers_success():
+    source_record = create_ead_source_record_stub(
+        metadata_insert=(
+            """
+            <repository>
+                <corpname>
+                    Massachusetts
+                    <emph>Institute</emph>
+                    of Technology. Libraries. Department of Distinctive Collections
+                </corpname>
+            </repository>
+            """
+        ),
+        parent_element="did",
+    )
+    assert Ead.get_publishers(source_record) == [
+        timdex.Publisher(
+            name=(
+                "Massachusetts Institute of Technology. Libraries. "
+                "Department of Distinctive Collections"
+            )
+        )
+    ]
+
+
+def test_get_publishers_transforms_correctly_if_fields_blank():
+    source_record = create_ead_source_record_stub(
+        metadata_insert=(
+            """
+            <repository></repository>
+            """
+        ),
+        parent_element="did",
+    )
+    assert Ead.get_publishers(source_record) is None
+
+
+def test_get_publishers_transforms_correctly_if_fields_missing():
+    source_record = create_ead_source_record_stub(parent_element="did")
+    assert Ead.get_publishers(source_record) is None
+
+
 def test_get_main_titles_success():
```

Comment:
Again to match the method order in `ead.py`, I'd put the `get_main_titles` tests after the `get_summary` tests

Reply:
Fixed!